packages/twenty-front/src/modules/ui/field/display/components/CurrencyDisplayWithIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import { useTheme } from '@emotion/react';
+
+import { SettingsObjectFieldCurrencyFormValues } from '@/settings/data-model/components/SettingsObjectFieldCurrencyForm';
+import { SETTINGS_FIELD_CURRENCY_CODES } from '@/settings/data-model/constants/SettingsFieldCurrencyCodes';
+
+export type CurrencyDisplayWithIconsProps = {
+  currencyValues: SettingsObjectFieldCurrencyFormValues | undefined;
+};
+
+export const CurrencyDisplayWithIcons = ({
```

Comment:
Let's merge this component with `CurrencyDisplay`:
- move the code from this component to `CurrencyDisplay`
- keep `CurrencyDisplay` only with all the needed logic.